src/coap_resource.c

```diff
@@ -152,6 +154,7 @@ coap_print_wellknown(coap_context_t *context, unsigned char *buf, size_t *buflen
     {0, NULL}
   };
 #endif /* WITHOUT_QUERY_FILTER */
+  (void)session;
```

Comment:
I don't think we should be calling `coap_print_wellknown()` (or` coap_print_wellknown_lkd()`) with session.